.gitignore

```diff
@@ -19,3 +19,4 @@
 /.vscode/
 /x/evm/client/cli/testdata/
 /node_modules/
+*.DS_Store
```

Comment:
I don't think this is supposed to be here?

Reply:
Well, Mac OS does add this cruft so would be nice to ignore it.